README.md

```diff
@@ -191,7 +191,7 @@ The migrations are stored in the `db/data` folder.
 
 - To generate a migration: `rails g data_migration add_this_to_that`
```

Comment:
@and-lucas-kelly being really pedantic here, but I think for extra clarity it should say `data migration`.
I know its in the topic area, but I have added this as on the line directly after we call it a `data migration`

Reply:
makes sense to me.